keyboards/crkbd/keymaps/nimishgautam/config.h

```diff
@@ -0,0 +1,100 @@
+// Copyright 2022 Nimish Gåtam (@nimishgautam)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// OS-specific stuff
+//#define UNICODE_SELECTED_MODES UC_MAC
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+//#define MASTER_LEFT
+#define MASTER_RIGHT
+// #define EE_HANDS
+#define SPLIT_LAYER_STATE_ENABLE
+#define SPLIT_MODS_ENABLE
+
+
+// combo
+#define COMBO_COUNT 7
+#define EXTRA_SHORT_COMBOS
+
+//Tapping values
+//#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 200
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD_PER_KEY
+
+
+// Try to save some memory
+#define NO_ACTION_ONESHOT
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
```

Comment:
These aren't needed.  If console isn't enabled, these are already defined (it's why they're checked first, actually). And if console is enabled ... you want these enabled. 
```suggestion
```

Reply:
Removed, they were pasted from an older attempt to save memory